test/vif.jl

```diff
@@ -0,0 +1,59 @@
+using GLM
+using LinearAlgebra
+using MixedModels
+using MixedModelsExtras
+using Test
+
+using MixedModels: dataset
+using RDatasets: RDatasets
+rdataset = RDatasets.dataset
```

Comment:
```suggestion
using RDatasets: dataset as rdataset
```
This should work since you require 1.6